### ISSUE TITLE
[Components] Bitrix24

### ISSUE THREAD

**Issue by dannyroosevelt** [PipedreamHQ/pipedream#5363]
### App
Bitrix24
https://training.bitrix24.com/rest_help/

**Triggers**:
1. New User - Triggers when a new user is added.
2. New Task - Triggers when a new task is created.

**Actions**:
1. Create Task - Creates a new task in Bitrix24.
2. Create User - Creates a new user in Bitrix24.
3. Update Task - Updates an existing task in Bitrix24.

**Comment by dylburger** [comment]
Thanks for the request! Did you know that anyone can [author and publish their own Pipedream actions](https://pipedream.com/docs/components/quickstart/nodejs/actions/) and [sources](https://pipedream.com/docs/components/quickstart/nodejs/sources/)? Would you be interested in helping out with that? We're happy to answer any questions or support you in any way.

If you don't have the time, would you be willing to reach out to the app provider to see if they're interested in building these? When we work with an app partner, it improves the quality of the integration, and it helps when this kind of request comes from a customer like you.

**Comment by luancazarine** [comment]
@dannyroosevelt It's needed to have a professional account
![image](https://user-images.githubusercontent.com/5638699/217271056-4bd73ea1-1809-4c9b-a683-499d40fa1140.png)

**Comment by michelle0927** [comment]
@dannyroosevelt Getting this error when trying to connect a Bitrix24 account to Pipedream.

<img width="595" alt="Screenshot 2023-02-15 at 1 43 32 PM" src="https://user-images.githubusercontent.com/10889229/219139050-a24bcd49-1aca-481f-8b67-20701f3b3a80.png">

**Comment by magnum79** [comment]
@luancazarine yes, it needs paid account, but you can create 15-days trial porfessional accounts as many times as you need and work with Bitrix24 REST API. Simply visit https://www.bitrix24.net/create/?from=profile
then press **CREATE BITRIX24** button and then **CREATE NEW BITRIX24** button.
New cloud instance will be created, and then you can enable 15-days trial, by pressing Application - Developer resources - Other - Local Application.
It will prompt you about Professional plan, but on the bottom of that window you will notice **TRY 15 DAYS FOR FREE** button
![image](https://github.com/PipedreamHQ/pipedream/assets/3918939/5670efad-25b7-48f2-963b-2a68adb307e0)

As experienced Bitrix24 user I can say - this is official way to create and test new Bitrix24 integrations for all their developers.

**Comment by magnum79** [comment]
@michelle0927 I wanted to use 'bitrix24' app in my pipedream workflow. [Here](https://github.com/PipedreamHQ/pipedream/issues/8487#issuecomment-1765357361) I found that it was existent in past, and was available at url https://pipedream.com/apps/bitrix24
But it's not anymore.
Here's Internet Archive capture for this page: https://web.archive.org/web/20231001092856/https://pipedream.com/apps/bitrix24

Seems app was removed from pipedream for some reason. If it was @luancazarine comment, please consider returning an app.

I want to have an ability to connect Bitrix24 account to my Pipedream and use it in my NodeJS steps.

If 'bitrix24' app had another issues, I could help resolving them. Even more - I could write my own app maybe. But first I'm asking you to restore deleted one. May be it works fine. Thanks!

**Comment by magnum79** [comment]
> @dannyroosevelt Getting this error when trying to connect a Bitrix24 account to Pipedream.
> 
> <img alt="Screenshot 2023-02-15 at 1 43 32 PM" width="595" src="https://user-images.githubusercontent.com/10889229/219139050-a24bcd49-1aca-481f-8b67-20701f3b3a80.png">

Yes, you will get this error unless you followed [these steps](https://training.bitrix24.com/support/training/course/?COURSE_ID=169&LESSON_ID=20184&LESSON_PATH=13643.20142.20178.20184) for creating so-called "GUI-less server local application"

Pay attention to paragraph

> Note the field Initial installation path. You have to enter the URL of the install.php you have extracted from the archive and uploaded to the server. This URL will be called only once when saving the local application form. The system uses this URL to send the [ONAPPINSTALL](https://training.bitrix24.com/rest_help/general/events/onappinstall.php), event the script uses to save authentication tokens of the current user. Whenever you want to update these tokens manually, open the Applications page and select the Re-install option.

Here you must supply some path at pipedream.com, that will receive an auth token. After that pipedream can authorize using this auth token.

Bitrix24 have published their [PHP SDK for working with their REST API](https://github.com/bitrix-tools/crest#contenten). You can look through that code to undersand how installation of newely created bitrix24 application works.
